tests/newSequences.mapp

```diff
@@ -0,0 +1,61 @@
+# name                    number of elements       address          size           bar    width   fracbits    signed
+
+# The sequences with default fixed point interpretation 32 bit int (also the shorter ones)
+TEST.MEM_MULTIPLEXED_INT 15 0 60 0
+TEST.MEM_MULTIPLEXED_INT.0 1 0 4 0
+TEST.MEM_MULTIPLEXED_INT.1 1 4 4 0
+TEST.MEM_MULTIPLEXED_INT.2 1 8 4 0
+
+# There are only 15 entries in the sequence, but the AREA has to be a multiple
+# of 4 bytes (32 bits) in order to work with the librabry and the drivers.
```

Comment:
This should not be the case any more. We should be able to do "strange" register sizes and the backend is supposed to do the closest possible transfer size to include all requested bytes.

Reply:
Sorry, I didn't realise this. Can you quickly try without the rounding to multiple of 4? It should work as well. If it still doesn't, create a ticket and change the comment to point to that ticket.